spec/models/driver_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe Driver do
+  # Rspec Documentation: https://github.com/rspec/rspec-rails
+  # how to run tests: bundle exec rspec
+  # how to run individual test: bundle exec rspec <file_path>
+  # copy the filepath from the explorer on the left.
+
+  describe 'driver validations' do
+    subject { driver.valid? }
+
+    let(:driver) { Driver.new(
+      name: name,
+      license_number: license_number,
+      mobile_number: mobile_number,
+      email_address: email_address,
+      age: age,
+      status: status,
+    ) }
```

Comment:
```suggestion
    let(:driver) do
      Driver.new(
        name: name,
        license_number: license_number,
        mobile_number: mobile_number,
        email_address: email_address,
        age: age,
        status: status,
      )
    end
```
Multi line curly brace blocks `{}` are frowned upon, its better to just use `do ... end` for clarity on what the block is

can we also change this to be a factory rather then using `Driver.new` we can use `build(:driver`